pattoo/api/portal/react/src/Login.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+class Login extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            username: '',
+            password: ''
+        };
+
+        this.handleInputChange = this.handleInputChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
```

Comment:
Where are the comments on what you are doing and why? This is important for future developers.